client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -29,8 +29,8 @@ float get_rgb_height (vec2 uv){
 
 vec4 get_normal_map(vec2 uv){
 	vec4 bump = texture2D(normalTexture, uv).rgba;
-	bump.xyz = normalize(bump.xyz * 2.0 -1.0);
 	bump.y = -bump.y;
+	bump.xyz = normalize(bump.xyz * 2.0 -1.0);
```

Comment:
`-` isn't used as unary operator here, and therefore has to be enclosed in spaces `* 2.0 - 1.0);`
Only a style nitpick.